src/Codeception/Lib/InnerBrowser.php

```diff
@@ -181,6 +191,9 @@ public function _getResponseContent()
         return (string)$this->getRunningClient()->getInternalResponse()->getContent();
     }
 
+    /**
+     * @return \Symfony\Component\DomCrawler\Crawler|mixed
```

Comment:
I checked it, there is no mixed, it only returns Crawler.

`redirectIfNecessary` method has incorrect `@return mixed` annotation, it returns only Crawler too.